src/clue/app-config.json

```diff
@@ -211,6 +211,11 @@
         "title": "Drawing",
         "isTileTool": true
       },
+      {
+        "id": "Starter",
+        "title": "Starter",
+        "isTileTool": true
+      },
```

Comment:
You could eliminate the need for this with a new curriculum unit so that the new tile only appeared for curricula that requested it.

Reply:
@kswenson what do you think about putting this in the "example" unit? Or do you think we need to make a whole new unit?

Reply:
Example unit seems fine (as you've now done).